liualgotrader/data/alpaca.py

```diff
@@ -33,11 +33,15 @@ def __init__(self):
                 "Failed to authenticate Alpaca RESTful client"
             )
 
-    def get_symbols(self) -> List[Dict]:
+    def get_symbols(self) -> List[str]:
         if not self.alpaca_rest_client:
             raise AssertionError("Must call w/ authenticated Alpaca client")
 
-        return self.alpaca_rest_client.list_assets()
+        data = self.alpaca_rest_client.list_assets(asset_class='us_equity')
+        return [_d.symbol for _d in data]
+
+    def get_market_snapshot(self) -> List[Dict]:
```

Comment:
@riven314 why is the function returning an empty list vs the snapshot? Maybe I'm missing something here.

Reply:
it was WIP by the time
just pushed a commit that implement this function